backend/lib/routes/info.js

```diff
@@ -20,38 +20,40 @@ const express = require('express')
 const got = require('got')
 const logger = require('../logger')
 const { decodeBase64 } = require('../utils')
-const kubernetes = require('../kubernetes')
+const { isHttpError } = require('../kubernetes-client')
 const { version } = require('../../package')
 
-const apiRegistration = kubernetes.apiRegistration()
-
 const router = module.exports = express.Router()
 
 router.route('/')
   .get(async (req, res, next) => {
     try {
-      const user = req.user
-      const gardenerVersion = await fetchGardenerVersion()
-      res.send({ version, gardenerVersion, user })
+      const gardenerVersion = await fetchGardenerVersion(req.user)
+      res.send({ version, gardenerVersion })
     } catch (err) {
       next(err)
     }
   })
 
-async function fetchGardenerVersion () {
+async function fetchGardenerVersion (user) {
+  const client = user.client // user specific client for the garden cluster
   try {
-    const { spec: { service, caBundle } } = await apiRegistration.apiservices.get({
```

Comment:
can this be called by a regular project member / user

Reply:
No. Even an admin is not authorized. Use dashboardClient.
✔️